### DISCUSSION TITLE
Anyone have a working example for a svelte virtualized table with expandable rows?

### DISCUSSION THREAD

**Discussion by dominictwlee** [TanStack/virtual#652] [Q&A]
It seems that virtualized tables will scroll to top if an expandable row is toggled. Does anyone know whether this is intentional, and whether there are alternative solutions to this?

Based this off the svelte example for virtualized table + adding simple expandable functionality with subrows.

**Comment by piecyk** [comment]
I think it's connected with our auto adjustment logic, when item is resizes https://github.com/TanStack/virtual/blob/main/packages/virtual-core/src/index.ts#L655-L667

In the end we only need it when scrolling backward to retain scroll position when element sizes are difirent from estimations

**Comment by dominictwlee** [comment]
Ah yeah, that seems to be it. Shall I raise an issue with a minimum repro then?

**Comment by dominictwlee** [comment]
Created an issue for this: https://github.com/TanStack/virtual/issues/656
